proto/proto/stake.proto

```diff
@@ -59,6 +59,13 @@ message ValidatorDefinition {
   bytes auth_sig = 2;
 }
 
+// Describes the current state of a validator on-chain
+message ValidatorStatus {
+  Validator validator = 1;
+  uint64 voting_power = 2;
+  // ... ?
+}
+
```

Comment:
This is a stub, but we should figure out what information we need to record about the state of a validator earlier rather than later.  The `voting_power` can move out of the `RateData` structure, since it's actually a different category of information (it's based on the delegations, not just the validator parameters).  We probably also want to record whether or not the validator is active, inactive, jailed, etc.